src/CMakeLists.txt

```diff
@@ -165,12 +165,20 @@ target_compile_definitions(digidocpp_priv PUBLIC
     $<$<CXX_COMPILER_ID:Clang>:XSD_CXX11_TEMPLATE_ALIAS>
 )
 
+target_include_directories(digidocpp_util PUBLIC
+    ${Iconv_INCLUDE_DIR}
+)
+
```

Comment:
Is this actually needed?
target_link_libraries(digidocpp_util Iconv::Iconv)
will include also include directories

Reply:
Ah yes, also works without.

I think its a left-over from my first diff which then stuck across the rebase from v3.14.8 onto latest HEAD.